projects/core/src/lib/core.module.ts

```diff
@@ -51,7 +50,7 @@ import { UserStoreFacade } from "./Store/users/users.store.facade";
 		HomePageComponent,
 		InfoPanelComponent
 	],
-	exports: [CoreComponent, HeaderComponent, FooterComponent, InfoPanelComponent],
+	exports: [CoreComponent, HeaderComponent, FooterComponent, InfoPanelComponent, ButtonComponent],
```

Comment:
ButtonComponent should not be exported from CoreModule

Reply:
done